keyboards/tr60w/config.h

```diff
@@ -0,0 +1,58 @@
+#ifndef CONFIG_H
```

Comment:
QMK uses the `#pragma once` include guard, instead of `#ifndef`/`#define`/`#endif`.

```suggestion
#pragma once
```

Reply:
done.